src/schemathesis/specs/openapi/parameters.py

```diff
@@ -0,0 +1,348 @@
+    examples_field = "examples"
+    nullable_field = "nullable"
+    # https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md#schema-object
+    # Excluding informative keywords - `title`, `description`, `default`
+    supported_jsonschema_keywords = (
+        "multipleOf",
+        "maximum",
+        "exclusiveMaximum",
+        "minimum",
+        "exclusiveMinimum",
+        "maxLength",
+        "minLength",
+        "pattern",
+        "maxItems",
+        "minItems",
+        "uniqueItems",
+        "maxProperties",
+        "minProperties",
+        "required",
+        "enum",
+        "type",
+        "allOf",
+        "oneOf",
+        "anyOf",
+        "not",
+        "items",
+        "properties",
+        "additionalProperties",
+        "format",
+    )
+
+    @property
+    def is_header(self) -> bool:
+        return self.location in ("header", "cookie")
+
+    def from_open_api_to_json_schema(self, open_api_schema: Dict[str, Any]) -> Dict[str, Any]:
+        open_api_schema = get_parameter_schema(open_api_schema)
+        return super().from_open_api_to_json_schema(open_api_schema)
+
+
+@attr.s(slots=True)
+class OpenAPIBody(OpenAPIParameter):
+    media_type: str = attr.ib()
+
+    @property
+    def location(self) -> str:
+        return "body"
+
+    @property
+    def name(self) -> str:
+        # The name doesn't matter, but is here for the interface completeness.
+        return "body"
+
+
+@attr.s(slots=True)
+class OpenAPI20Body(OpenAPIBody, OpenAPI20Parameter):
+    """Open API 2.0 body variant."""
+
+    def as_json_schema(self) -> Dict[str, Any]:
+        """Convert body definition to JSON Schema."""
+        # `schema` is required in Open API 2.0 bodies.
+        schema = self.definition["schema"]
+        return self.transform_keywords(schema)
+
+
+@attr.s(slots=True)
+class OpenAPI30Body(OpenAPIBody, OpenAPI30Parameter):
+    """Open API 3.0 body variant.
+
+    We consider each media type defined in the schema as a separate variant, that can be chosen for generation.
+    The value of the `definition` field is essentially the Open API 3.0 `MediaType`.
+    """
+
+    # `required` keyword is located above the schema for concrete media-type; therefore it is passed here explicitly
+    required: bool = attr.ib(default=False)
+
+    def as_json_schema(self) -> Dict[str, Any]:
+        """Convert body definition to JSON Schema."""
+        schema = get_media_type_schema(self.definition)
+        return self.transform_keywords(schema)
+
+    def transform_keywords(self, schema: Dict[str, Any]) -> Dict[str, Any]:
+        definition = super().transform_keywords(schema)
+        if self.is_form:
+            definition.setdefault("type", "object")
+        return definition
+
+    @property
+    def is_form(self) -> bool:
+        """Whether this payload represent a form."""
+        return self.media_type in ("multipart/form-data", "application/x-www-form-urlencoded")
```

Comment:
Are there any other media types for this purpose? Also, it could be a module-level constant